pkg/resources/digitalocean/cloud.go

```diff
@@ -38,6 +39,8 @@ import (
 
 const TagKubernetesClusterIndex = "k8s-index"
 const TagKubernetesClusterNamePrefix = "KubernetesCluster"
+const TagKubernetesInstanceGroup = "kops-instancegroup"
```

Comment:
Why have a duplicate of `do.TagKubernetesClusterInstanceGroupPrefix`?

Reply:
@johngmyers  - I think this needs refactoring. I tried to use it and it looks like we have a cyclic dependency here. Ideally, I should have kept this logic (and few other methods) in the do package, but that's a little more work that I thought of.
Would it be okay if we get this merged, and I take up the clean up activity as a separate PR? Please suggest.

Reply:
Perhaps move `TagKubernetesClusterInstanceGroupPrefix` here?

Reply:
I removed TagKubernetesClusterInstanceGroupPrefix and instead used the above const.

Reply:
Resolved